rebasehelper/spec_hooks/pypi_url_fix.py

```diff
@@ -53,14 +56,25 @@ def run(cls, spec_file, rebase_spec_file, **kwargs):
         """
         for index, line in enumerate(rebase_spec_file.spec_content):
             if line.startswith("Source"):
+                rebase_spec_file.spec_content[index] = cls._transform_sources_url(line)
+            elif line.startswith("URL"):
                 rebase_spec_file.spec_content[index] = cls._transform_url(line)
         rebase_spec_file.save()
 
     @classmethod
-    def _transform_url(cls, line):
+    def _transform_sources_url(cls, line):
         """
-        Perform predefined URL transformations
+        Perform predefined URL transformations to sources line.
```

Comment:
What is `sources line`? It should be `on Source line` probably.

Reply:
But the docstring is probably redundant in this case, the method is short and it's obvious what it does.